docs/querying/sql-functions.md

```diff
@@ -697,11 +744,26 @@ Returns the value of a numeric or string expression corresponding to the earlies
 
 ## EXP
 
-`EXP(<NUMERIC>)`
+Calculates _e_ raised to the power of the numeric expression.
 
-**Function type:** [Scalar, numeric](sql-scalar.md#numeric-functions)
+* **Syntax:** `EXP(<NUMERIC>)`
+* **Function type:** Scalar, numeric
 
-Calculates _e_ raised to the power of the numeric expression.
+<details><summary>Example</summary>
+
+The following example calculates _e_ to the power of 2.
```

Comment:
It looks like e^1 not e^2. Did you mean to change the example?

Reply:
Thank you for catching it, I've corrected it